packages/react-meteor-data/useFind.ts

```diff
@@ -144,6 +145,10 @@ const useFindServer = <T = any>(factory: () => Mongo.Cursor<T> | undefined | nul
   Tracker.nonreactive(() => {
     const cursor = factory()
     if (Meteor.isDevelopment) checkCursor(cursor)
+    // in the future this maybe the default behavior
+    // for now we will use the suspense option to enable this
+    const [{suspense, suspenseLifespan}] = deps;
+    if (suspense) return usePromise(() => cursor?.fetchAsync?.(), [], suspenseLifespan || 0) ?? null
```

Comment:
1. It should be a separate, optional argument, after the `deps`. With this implementation, `[undefined]` in `deps` will break the app.
2. What is `suspenseLifespan` for? I see that it behaves like a timeout on the promise, but I wouldn't add it now and focus on testing the `fetchAsync` alone.